parasolr/solr/tests/test_schema.py

```diff
@@ -26,19 +26,19 @@ def test_delete_field(self, test_client):
     def test_replace_fields(self, test_client):
 
         test_client.schema.add_field(name='A', type='string')
-        test_client.schema.replace_field(name='A', type='int')
+        test_client.schema.replace_field(name='A', type='pint')
```

Comment:
Can you provide a link to help me understand this change? I'm not finding much when I search for it.

Reply:
thank you for explaining! what a pain